src/main/java/tetris/genetic/GeneticAlgorithm.java

```diff
@@ -31,18 +31,16 @@
 public class GeneticAlgorithm {
 
   private static final Integer NUM_GENERATIONS = 100;
-  private static final Integer NUM_GAMES = 50;
+  private static final Integer NUM_GAMES = 10;
+  private static final Integer POPULATION_SIZE = 200;
 
   private static final Double MUTATION_RATE = 0.1;
-  private static final Integer SELECTION = 5;
   private static final Double DEFAULT_SCORE = 0.0;
 
   private static final Integer SURVIVORS = 15;
```

Comment:
Are we still leaving so many survivors from the last round? Would something like 5 work better?